src/main/java/com/rainbowrave/Theme.java

```diff
@@ -0,0 +1,97 @@
+package com.rainbowrave;
+
+import java.awt.Color;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public enum Theme
+{
+    DEFAULT,
+    TRANS(new Color[]{
+            new Color(91, 206, 250),
+            new Color(245, 169, 184),
+            new Color(255, 255, 255),
+            new Color(245, 169, 184),
+            new Color(91, 206, 250)
+    }),
+    ENBY(new Color[]{
+            new Color(255, 255, 255),
+            new Color(156, 89, 209),
+            new Color(44, 44, 44),
+            new Color(252, 244, 52),
+            new Color(255, 255, 255),
+    }),
+    LESBIAN(new Color[]{
+            new Color(212, 44, 0),
+            new Color(253, 152, 85),
+            new Color(255, 255, 255),
+            new Color(209, 97, 162),
+            new Color(162, 1, 97),
+            //new Color(212, 44, 0)
```

Comment:
The colors were based on the different flags and the ordering there. Like the trans flag has a natural wrapping in its ordering. 
![image](https://user-images.githubusercontent.com/85463994/184030887-921056d2-268c-42d5-a3e8-88b23f3ab48e.png)


I wrapped the themes when the first and last colors were very different and it would create a flickering effect after an otherwise smooth transition. Also if a specified color was contained in the gradient of another colors' transition it would create a bias towards that color. But in general, I was just looking at them and deciding what I thought looked best.

Reply:
ah, all interesting points. It does seem though that the constructor could automatically attach the last color to the first color rather than requiring almost every theme to require duplicate start and end colors. Is this reasonable?

Reply:
Yeah, that totally works for me.